### DISCUSSION TITLE
What do the letters in lualine + noice mean?

### DISCUSSION THREAD

**Discussion by aguynamedben** [LazyVim/LazyVim#4373] [Q&A]
LazyVim's lualine setup ([docs](https://www.lazyvim.org/plugins/ui#lualinenvim)) in `lualine_x` we have stuff from [noice.nvim](https://github.com/folke/noice.nvim). It seems this is used to show the LSP starting and other messages, which is good.

Question: When editing a file this area of lualine is being updated in a very distracting way, usually showing "g" followed by whatever normal mode key I just hit.

* What do the letters mean?
* What is the reason for wanting to see it here?
* How can I hide it? What do I lose out on if I hide it?

I looked at Noice for a while, but it seems kinda crazy, and I don't see anything in the LayzVim lualine config that really explains why we might want to see these cryptic letters here!

**Comment by dpetka2001** [comment]
The letters are just the keys that you pressed. If you press `gg`, it will show `gg` in the statusline and so on. If you don't want to see that you can do
```lua
{
  "nvim-lualine/lualine.nvim",
  opts = function(_, opts)
    table.remove(opts.sections.lualine_x, 1)
  end,
},
```
